src/main/kotlin/no/nav/familie/tilbake/oppgave/OppgaveService.kt

```diff
@@ -39,17 +42,18 @@ class OppgaveService(private val behandlingRepository: BehandlingRepository,
         val behandling = behandlingRepository.findByIdOrThrow(behandlingId)
         val fagsakId = behandling.fagsakId
         val fagsak = fagsakRepository.findByIdOrThrow(fagsakId)
+        val aktorId = personService.hentAktivAktørId(fagsak.bruker.ident, fagsak.fagsystem)
```

Comment:
Trodde vi skulle legge in denne koden i OppgaveService i familie-integrasjoner jeg...

Reply:
Det var planen ja :), men synes det ble feil å ha en kobling mot PDL tema i oppgaveservice.